src/dataprep/main/link/read_topic_similarity.py

```diff
@@ -78,10 +80,10 @@
 
 
 for id, params in file_map.items():
-    subprocess.run(f"tail -n +2 -q {args.read_dir}/{id}-part-*.csv >> {params['fn_full']}", shell=True)
+    subprocess.run(f"tail -n +2 -q {args.read_dir}/{id}-maxlevel-*-part-*.csv >> {params['fn_full']}", shell=True)
     with con as c:
-        c.execute(f"DROP TABLE IF EXISTS {params['tbl']}")
-        c.execute(f"CREATE TABLE {params['tbl']} {params['schema']}")
+         c.execute(f"DROP TABLE IF EXISTS {params['tbl']}")
```

Comment:
should this be changed to drop table only if option is set to delete it, otherwise append?

Reply:
I did not change anything substantial in this part (yet). Instead of appending to existing tables, I concatenate all csv files in the step before (line 83 now). this was easier to change. it means we cannot add similarity measures at different levels later on, but I thought we can just run all max levels at once and then we're done with it.

Reply:
I like this solution! Sorry missed that.